language/move-prover/doc/user/prover-guide.md

```diff
@@ -1,59 +1,144 @@
-# Move Prover User Guide (DRAFT)
+# Move Prover User Guide
 
 This is the user guide for the Move prover. This document does not describe the
 [Move specification language](spec-lang.md), but accompanies it.
 
-This guide is currently specific for the usage of the prover within the Diem repo.
+- [Running the Prover](#running-the-prover)
+    - [Target Filtering](#target-filtering)
+    - [Prover Options](#prover-options)
+    - [Prover Configuration File](#prover-configuration-file)
+    - [Prover Tests](#prover-tests)
+- [Prover Diagnosis](#prover-diagnosis)
+    - [Unexpected Abort](#unexpected-abort)
+    - [Postcondition Failure](#postcondition-failure)
+- [Debugging the Prover](#debugging-the-prover)
 
 ## Running the Prover
 
-To run the Move prover while working in the Diem tree, we recommend to use `cargo run`. You can define an alias
-as below in your `.bashrc` (or other shell configuration) to simplify this:
+The prover is invoked via the Move CLI. When working in a repo which contains the CLI, you can make it available via an
+alias as below:
 
 ```shell script
-alias mvp="cargo run --release --quiet --package move-prover --"
+alias move="cargo run --release --quiet --package move-cli --"
 ```
 
-The `--release` flag can also be omitted if you prefer faster compilation over faster execution. However, once the
-prover has been build by cargo, and provided sources have not changed, the cargo command will quickly proceed.
-Note that the Rust code of the prover (and the underlying Move compiler) is by about a factor of 20 faster in
-release mode than in debug mode.
+> NOTE: The `--release` flag can also be omitted if you prefer faster compilation over faster execution. Note, however,
+> that the Rust code part of the prover (and the underlying Move compiler) is by an order of magnitude faster
+> in release mode than in debug mode.
 
-In the sequel, we assume that `mvp` invokes the binary of the prover, either via an alias as above, or by
-other means.
+We assume in the sequel that the Move CLI is reachable from the command line via the `move` command
+(defined by an alias as above or by other means).
 
-## Command Line Interface
+In order to call the CLI, you must have a [*move package*](TODO). In the simplest case, a Move package is defined by a
```

Comment:
Nit: you can probably point this ref to here: https://diem.github.io/move/packages.html

Reply:
Done